src/prefect/indicators/run.py

```diff
@@ -0,0 +1,4 @@
+"""QualiCharge prefect indicators: entrypoint."""
+
+# ruff: noqa: F401
+from .infrastructure.i1 import calculate as i1_calculate
```

Comment:
To avoid to change code for each addition of a new indicator, we could have all ixx_calculate functions in the same module and use getattr : 

```python
import calculate_module

indicator = 'i1'
indicator_calculate = getattr(calculate_module, indicator + '_calculate')
```

Reply:
Yes, we'll come with something more elegant (automatized) in another PR.